apps/emqx_connector_aggregator/src/emqx_connector_aggreg_delivery.erl

```diff
@@ -25,14 +25,16 @@
 ]).
 
 -record(delivery, {
-    name :: _Name,
+    id :: aggreg_id(),
```

Comment:
Nit. This reads like _delivery id_ rather than _which aggregator this delivery was spawned by_.

Reply:
Sorry for not being clear enough: I meant that `#delivery.id` sounds like it's something that should uniquely identify a delivery. But it's actually not: there can be multiple deliveries with the same `#delivery.id`.

But it's just a nit after all, not that important.

Reply:
Aha, sorry, I missed the point.  :sweat_smile: 

I'll fix the field and binding names in a follow up then.  Thanks!